admin/view-translations-post.php

```diff
@@ -47,8 +49,11 @@
 			 * The dynamic portion of the hook name, `$lang`, refers to the language code.
 			 *
 			 * @since 2.1
+			 * @since 3.8 New `$post_type` parameter.
+			 *
+			 * @param string $post_type Post's post type.
```

Comment:
```suggestion
			 * @param string $post_type The post type.
```
Same suggestion as above.